mail-filter/mimedefang/mimedefang-2.84.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit user
+
+DESCRIPTION="Antispam, antivirus and other customizable filters for MTAs with Milter support"
+HOMEPAGE="http://www.mimedefang.org/"
+SRC_URI="http://www.mimedefang.org/static/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="clamav +poll test"
+
+DEPEND=">=dev-perl/MIME-tools-5.412
+	dev-perl/IO-stringy
+	virtual/perl-MIME-Base64
+	dev-perl/Digest-SHA1
+	dev-perl/MailTools
+	dev-perl/Unix-Syslog
+	clamav? ( app-antivirus/clamav )
+	|| ( mail-filter/libmilter mail-mta/sendmail )
+	test? (
+		dev-perl/Test-Class
+		dev-perl/Test-Most
+	)"
+RDEPEND="${DEPEND}"
+RESTRICT="test? ( !test )"
```

Comment:
You got it the other way around. I'm surprised Portage doesn't reject it as invalid request.